airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceDatatypeTest.java

```diff
@@ -82,4 +89,288 @@ public boolean testCatalog() {
     return true;
   }
 
+  @Override
+  protected void initTests() {
+    super.initTests();
+    addArraysTestData();
+  }
+
+  private void addArraysTestData() {
```

Comment:
These tests be added in AbstractPostgresSourceDatatypeTest so that they run for CDC mode as well.

Reply:
> These tests should be added in AbstractPostgresSourceDatatypeTest so that they run for CDC mode as well.

updated tests for CDC